src/Apps/Loyalty/Containers/AcbThankYou/index.tsx

```diff
@@ -1,14 +1,14 @@
 import * as React from "react"
 import styled from "styled-components"
 
-import colors from "../../../../assets/colors"
-
-import Button from "../../../../components/buttons/inverted"
-import Nav from "../../../../components/nav"
-import NavItem from "../../../../components/nav_item"
-import Text from "../../../../components/text"
-import TextLink from "../../../../components/text_link"
-import Title from "../../../../components/title"
+import colors from "../../../../Assets/Colors"
+
+import Button from "../../../../Components/Buttons/Inverted"
+import Nav from "../../../../Components/Nav"
+import NavItem from "../../../../Components/NavItem"
+import Text from "../../../../Components/Text"
+import TextLink from "../../../../Components/TextLink"
+import Title from "../../../../Components/Title"
```

Comment:
@alloy - Curious: Why capitalize path fragments? I've never really seen that before, where file case is common since that helps keeping things 1-1:
```js
import Title from 'foo/bar/Title'
import someUtil from 'foo/bar/baz/someUtil'
```

Reply:
Good question, alas I don’t recall what blog/SO post I eventually settled on, but that’s what I did, no other tonal of my own :/

Reply:
Yeah you make great points! 🤔

Sigh, not looking forward to redoing that 😁, but I’ll redo that during this migration 👍

Reply:
Thanks @alloy 👍